jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -43,5 +43,22 @@
                 'log_reference': 'log_info.log',
             },
         },
+        'media-default': {
```

Comment:
I would call it `media-tests` and add another suffix for all the other extended test types, eg. `media-tests-youtube`.